### PR TITLE
Footer component implementation

### PR DESCRIPTION
**Pull request by DarinaBlazevica**
-Create Footer Component(+styling)
-Create SocialMedia Component(+styling)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,11 +1,13 @@
 import "./App.css";
 import About from "./components/organisms/About/About";
+import Footer from "./components/organisms/Footer/Footer";
 import Gallery from "./components/organisms/Gallery/Gallery";
 import structure from "./structure.json";
```
Comment (ilze-dom) [inline src/App.tsx]: This should not be a direct import. Use js fetch API

Comment (DarinaBlazevica) [reply src/App.tsx]: Data fetch added

File: src/App.tsx [hunk 2]
```diff
@@ -1,25 +1,53 @@
...
 import "./App.css";
 import About from "./components/organisms/About/About";
+import Footer from "./components/organisms/Footer/Footer";
 import Gallery from "./components/organisms/Gallery/Gallery";
-import structure from "./structure.json";
 
 const App = () => {
-  const about = structure.about;
-  const gallery = structure.gallery;
+  const [data, setData] = useState<any>();
```
Comment (Jalal999) [inline src/App.tsx]: 'any' type for useState is not appropriate. try to come up with your own data type

Comment (DarinaBlazevica) [reply src/App.tsx]: I've came up with solution, hope this will work, if not will proceed with changes again

File: src/App.tsx [hunk 3]
```diff
@@ -1,25 +1,59 @@
...
 import About from "./components/organisms/About/About";
+import Footer from "./components/organisms/Footer/Footer";
 import Gallery from "./components/organisms/Gallery/Gallery";
-import structure from "./structure.json";
 
 const App = () => {
-  const about = structure.about;
-  const gallery = structure.gallery;
+  type JSONData = {
+    about: any
```
Comment (ilze-dom) [inline src/App.tsx]: please create a proper interface and don't use any in this case

File: src/App.tsx [hunk 4]
```diff
@@ -5,7 +5,13 @@ import Footer from "./components/organisms/Footer/Footer";
 import Gallery from "./components/organisms/Gallery/Gallery";
 
 const App = () => {
-  const [data, setData] = useState<any>();
+  type JSONData = {
+    about: any
+    gallery: any
+    footer: any
```
Comment (Jalal999) [inline src/App.tsx]: why any? you have data types already implemented in your components. import them and assign accordingly

Comment (DarinaBlazevica) [reply src/App.tsx]: Changed

**Review by Jalal999 — APPROVED** [review]

**Review by ilze-dom — APPROVED** [review]

File: src/App.tsx (hunk 3 above)
Comment (DarinaBlazevica) [reply src/App.tsx]: solved

**Pull request merged**